### ISSUE TITLE
paths incorrect in "when" stmts VLAN model

### ISSUE THREAD

**Issue by alpesh-arrcus** [openconfig/public#85]
Several "when" statements in the openconfig-vlan.yang need to be udpated. The diff is below:

````
116c116
<       when "interface-mode = 'TRUNK'" {
---
>       when "../interface-mode = 'TRUNK'" {
131c131
<       when "interface-mode = 'ACCESS'" {
---
>       when "../interface-mode = 'ACCESS'" {
144c144
<       when "interface-mode = 'TRUNK'" {
---
>       when "../interface-mode = 'TRUNK'" {
370c370
<     when "oc-if:type = 'ift:ethernetCsmacd'" {
---
>     when "../oc-if:config/oc-if:type = 'ift:ethernetCsmacd'" {
382c382
<     when "oc-if:type = 'ift:ieee8023adLag'" {
---
>     when "../oc-if:config/oc-if:type = 'ift:ieee8023adLag'" {
391c391
<     when "oc-if:type = 'ift:l3ipvlan'" {
---
>     when "oc-if:config/oc-if:type = 'ift:l3ipvlan'" {

````

Please help address in next release. 

thanks

**Comment by LeonGWang** [comment]
Hi, I am running into the same issue. Wondering if a fix is in the works.

**Comment by LeonGWang** [comment]
I believe this is fixed in commit 5dc792b
